UKHomeOffice/sgmr-service
# NMSW-44 (Change help link to /voyage-plans/start)

### AC / Description of changes

After testing it was suggested that we change the link to Voyage plans to a link to the start page

### To Test

- Run app locally
- Go to localhost:8080
- Sign in
- Click the help link in the footer
- Click on the voyage plans link
- > You should be taken to /voyage-plans/start and should see information about voyage plans

### Notes

---

**Remember to merge to feature branches not `master`.**
